gvsbuild/utils/builder.py

```diff
@@ -230,13 +231,12 @@ def __check_tools(self, opts):
         log.start("Checking msys tool")
         msys_path = opts.msys_dir
         if not msys_path or not Path.exists(msys_path):
-            msys_paths = [
-                Path(r"C:\msys64"),
-                Path(r"C:\msys32"),
-                Path(r"C:\tools\msys64"),
-                Path(r"C:\tools\msys32"),
-            ]
-            for path in msys_paths:
+            # list of drive letters to check
+            possible_drives = ['%s:' % d for d in ascii_uppercase if os.path.exists('%s:' % d)]
```

Comment:
Hi @raikoug, thanks for the contribution! Instead of searching through 26 possible drives, what if we did C, D, and E? We are only trying to search in common locations, otherwise users should use the `--msys-dir` option to specify the location. I would think that high 90% of users have MSYS2 on their C drive, and then if we add a few more drive letters we are up to 99.9%.

Reply:
Hi, Sorry for late reply.
I think `os.path.exists('%s:' % d)` is not too heavy, the second part searching for the actually msys executable in the generated paths could? 
Otherwise in a normal situation with 4 drives letters active we would have 16 paths in `all_possible_paths`
I did not think about performance to be honest